ivy/functional/frontends/paddle/tensor/search.py

```diff
@@ -43,3 +43,21 @@ def nonzero(input, /, *, as_tuple=False):
     if as_tuple is False:
         ret = ivy.matrix_transpose(ivy.stack(ret))
     return ret
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float32", "float64", "int32", "int64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def searchsorted(
+    sorted_sequence, values, /, *, out_int32=False, right=False, name=None
```

Comment:
Also, we don't add `/, *` in the frontend functions signatures. We just copy the signature of the original function as is.